### PR TITLE
Clean authors, layout & co

### PR DESCRIPTION
**Pull request by zebuline**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -2,9 +2,11 @@
 Authors & contributors
 ######################
 
-Maintainer: Benoît Bryon <benoit@marmelune.net>, as a member of the
+Author: Benoît Bryon <benoit@marmelune.net>, as former member of the
 `PeopleDoc`_ team: https://github.com/peopledoc/
 
+Mainteners: `PeopleDoc`_ team: https://github.com/peopledoc/
```
Comment (brunobord) [inline AUTHORS]: Maintainers

File: AUTHORS [hunk 2]
```diff
@@ -2,9 +2,11 @@
 Authors & contributors
 ######################
 
-Maintainer: Benoît Bryon <benoit@marmelune.net>, as a member of the
+Author: Benoît Bryon <benoit@marmelune.net>, as former member of the
```
Comment (brunobord) [inline AUTHORS]: "as a former"

**Comment by zebuline** [comment]
@brunobord @yanoz OPM ?

**Comment by yanoz** [comment]
OPM

**Pull request merged**